### PR TITLE
Pin dependencies in Pipfile, and restore working Pyup bot updates by ignoring Pipfile.lock

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt thoughts?

Passing ad-hoc here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/stubattribution.adhoc/204/console

Built and ran locally, too.

### CODE REVIEW

**Comment by stephendonner** [comment]
Sorry for the missing context, and technically-incomplete PR, to match.

I've grown weary of playing whack-a-mole with some successful @pyup-bot PRs (https://github.com/mozilla/stubattribution-tests/pull/122) and others, like https://github.com/mozilla/stubattribution-tests/pull/120, which I recreated and submitted, but with the bumped meta hash.

@davehunt do you have a suggestion/preference for the balance of restoring @pyup-bot mergeable PR support, and yet still continuing with the Pipfile?  Is that possible, without removing ```--deploy``` and thus obviating the security/trust we get from the updated-hash Pipfile.lock?

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM python:2.7-alpine
 WORKDIR /src
 RUN pip install pipenv
 COPY Pipfile Pipfile.lock /src/
```
Comment (davehunt) [inline Dockerfile]: We should probably remove the Pipfile.lock if we're not using it here.

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM python:2.7-alpine
...
-RUN pipenv install --system --deploy
+RUN pipenv install --system --skip-lock --deploy
```
Comment (davehunt) [inline Dockerfile]: I think `--deploy` only makes sense when we have a Pipfile.lock

File: Dockerfile (hunk 1 above)
Comment (stephendonner) [reply Dockerfile]: Fixed.

File: Dockerfile (hunk 2 above)
Comment (stephendonner) [reply Dockerfile]: Fixed.

**Comment by stephendonner** [comment]
@davehunt should be good to go, now!

**Pull request merged**